docs/Design-and-Modeling/02-directory-structure.md

```diff
@@ -1,6 +1,6 @@
 # Project Structure (Design & Modeling)
 
-![Project Structure](../../assets/images/directory-structure-design.png)
+![](../../assets/images/directory-structure-design.png)
```

Comment:
Alt tags are helpful for screen readers / accessibility - suggest adding something back for this (doesn't have to be Project Structure if that doesn't make sense to you).

Reply:
Alt tags are only useful when they are adding something useful. When they are repeating a header, or doing anything else redundant, they're actually more annoying than having nothing there. If I were to try and explain the content of this image, I would be repeating information that is listed below, so instead of doing anything redundant I am simply making the image invisible to screen-readers.